spring/src/main/java/org/axonframework/spring/config/annotation/SpringContextHandlerDefinitionBuilder.java

```diff
@@ -31,6 +31,7 @@
  * @author Milan Savic
  * @since 3.3
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.